src/react-view/contents/rules/rule-list.tsx

```
@@ -0,0 +1,78 @@
+                </Box>
+            </Box>
+            {rule.children.length > 0 ? <RuleSubList rules={rule.children} /> : undefined}
+        </Box>
+    );
+};
+
+const RuleSubList: React.FC<{ rules: RuleEnriched[] }> = ({ rules }) => {
+    return (
+        <Box pad="small">
+            {rules.map(rule => (
+                <RuleSummary rule={rule} />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)